packages/devtools/src/sourceMap/loadOriginalSourceLoc.ts

```diff
@@ -33,7 +33,19 @@ function parseSourceUrl(sourceUrlWithLoc: string): MappedPosition | null {
   const paths = sourceUrlWithLoc.split(':');
   const column = Number(paths.pop());
   const line = Number(paths.pop());
-  const source = paths.join(':');
+
+  let source = paths.join(':');
+  try {
```

Comment:
nit: you could pull it to a separate function like `normalizeWebpackRelativeUrl` and then document + test

Reply:
The test won't work expected unfortunately as chrome has a [bug](https://bugs.chromium.org/p/chromium/issues/detail?id=869291) when parsing url from unknown origin.
the `pathname` is different between chrome and node:
- chrome 
  <img width="679" alt="Screenshot 2022-09-06 at 11 19 55" src="https://user-images.githubusercontent.com/28751745/188598320-96bff6be-e220-4043-b32e-cb9cf7ddfdf1.png">
- node 
  <img width="336" alt="Screenshot 2022-09-06 at 11 20 27" src="https://user-images.githubusercontent.com/28751745/188598345-ada7fba9-56ab-49e6-8874-81d00d5ead87.png">

Reply:
Ouch, yeah thanks for the info